### PR TITLE
Allow setElementCollisionsEnabled for buildings

### PR DESCRIPTION
**Pull request by TheNormalnij**
Test:
```
crun o = createBuilding(1337, 0, 0, 5, 0,0,0)
crun setElementCollisionsEnabled(o, false)
```

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks good 🚀

File: Client/mods/deathmatch/logic/CClientBuilding.h [hunk 1]
```diff
@@ -71,6 +73,7 @@ class CClientBuilding : public CClientEntity
     CVector    m_vPos;
     CVector    m_vRot;
     uint8_t    m_interior;
+    bool       m_bUsesCollision;
```
Comment (botder) [inline Client/mods/deathmatch/logic/CClientBuilding.h]: Rename this to `m_usesCollision`

File: Client/mods/deathmatch/logic/CClientBuilding.cpp [hunk 2]
```diff
@@ -115,6 +125,10 @@ void CClientBuilding::Create()
 
     m_pBuilding = g_pGame->GetPools()->GetBuildingsPool().AddBuilding(this, m_usModelId, &m_vPos, &vRot4D, m_interior);
 
+    if (m_bUsesCollision == false)
```
Comment (botder) [inline Client/mods/deathmatch/logic/CClientBuilding.cpp]:
```suggestion
    if (!m_bUsesCollision)
```

File: Client/mods/deathmatch/logic/CClientBuilding.cpp [hunk 3]
```diff
@@ -102,6 +103,15 @@ void CClientBuilding::SetModel(uint16_t model)
     }
 }
 
+void CClientBuilding::SetUsesCollision(bool state)
+{
+    m_bUsesCollision = state;
```
Comment (botder) [inline Client/mods/deathmatch/logic/CClientBuilding.cpp]: Add please
```cpp
if (m_bUsesCollision == state)
    return;
```

**Review by botder — CHANGES_REQUESTED** [review]

**Comment by TheNormalnij** [comment]
@botder ty

**Review by botder — APPROVED** [review]

**Pull request merged**